src/diagnostic-channel-publishers/src/redis.pub.ts

```diff
@@ -48,7 +48,7 @@ const redisPatchFunction: PatchFunction = (originalRedis) => {
 };
 
 export const redis: IModulePatcher = {
-    versionSpecifier: ">= 2.0.0 < 3.0.0",
+    versionSpecifier: ">= 2.0.0 < 4.0.0",

```

Comment:
You should modify `diagnostic-channel-publishers` devpendency package.json to take `3.x` for tests

Reply:
Modified